static/src/stylesheets/module/content/_photo-essay.scss

```diff
@@ -0,0 +1,93 @@
+.content--article {
+    .element-image--photo-esssay {
+        padding-bottom: 0;
+        margin-bottom: 0.5rem;
+
+        &.fig--no-caption.fig--has-shares {
+            padding-bottom: 0px !important;
+        }
+
+        .block-share {
+            display: none !important;
```

Comment:
The first !important is because it already gets given !important from somewhere else. The second one I think I can get rid of.